__tests__/Config.test.js

```diff
@@ -5,7 +5,11 @@ const OLD_ENV = process.env
 describe('Config', () => {
   describe('static default values', () => {
     it('sinopia uri has static value', () => {
-      expect(Config.sinopiaUri).toEqual('https://sinopia.io')
+      expect(Config.sinopiaUri).toEqual('sinopia.io')
```

Comment:
Minor feedback: having a `sinopiaUrl` and a `sinopiaUri` may be confusing in the future, especially to tired eyes failing to see the difference between the `i` and `l` (the contrast between them depends on the font). Would you consider renaming this `sinopiaDomainName`?